langchain/src/document_loaders/fs/unstructured.ts

```diff
@@ -51,6 +52,8 @@ export class UnstructuredLoader extends BaseDocumentLoader {
 
   private apiKey?: string;
 
+  private strategy?: string;
```

Comment:
I think you need to actually set this in the constructor:
`this.strategy = optionsOrLegacyFilePath.strategy ?? "hi_res"`